travis.cfg

```diff
@@ -1,35 +1,44 @@
 [buildout]
-extends = 
+extends = development.cfg
+
+[remotes]
+collective = https://github.com/collective
+plone = https://github.com/plone
+plonegovbr = https://github.com/plonegovbr
+simplesconsultoria = https://github.com/simplesconsultoria
+
+[buildout:os.environ.get('PRODUCTION')]
+extends =
```

Comment:
@idgserpro isso aqui sobrescreve o anterior se a variável `PRODUCTION` estiver setada.

Reply:
Ah, então você pode então apenas remover [o segundo sed](https://github.com/plonegovbr/portal.buildout/pull/89/files#diff-354f30a63fb0907d4ad57269548329e3L37). Recomendo também adicionar `# Se for pegar produção, extends é sobrescrito pra pegar apenas produção.`.

Reply:
isso tem que se descrito no documento de atualização da versão.